back-end/api/controllers/projects.controller.js

```diff
@@ -30,7 +30,7 @@ export default class ProjectsController {
    * @returns {Promise<void>}
    */
   static async apiDeleteProject(req, res, next) {
-    const projectName = req.body.project_name;
+    const projectName = req.query.project_name;
```

Comment:
are you sure this function works with query? I think I changed it to body because it wasn't working, I could be wrong though

Reply:
I changed it cuz i was having trouble with testing the API's on Postman. Do you know how to pass body params on Postman?

Reply:
Yep I can show you on a call later!!